src/service/SecureKeychain.ts

```diff
@@ -34,14 +38,14 @@ async function getSupportedBiometryType() {
   return Keychain.getSupportedBiometryType();
 }
 
-async function resetGenericPassword() {
+async function resetGenericPassword(service: string) {
   mmkvStorage.delete(SELECTED_ACCESS_CONTROL);
 
-  return Keychain.resetGenericPassword({service: defaultOptions.service});
+  return Keychain.resetGenericPassword({service});
 }
 
-async function getGenericPassword() {
-  const userCredentials = await Keychain.getGenericPassword();
+async function getGenericPassword(accountId: string) {
+  const userCredentials = await Keychain.getGenericPassword({service: getKeychainService(accountId)});
```

Comment:
I think we should keep this service (SecureKeychain) generic, later we could use it as well to unlock the entire app for example. `getGenericPassword` could take a `service` string instead of the `accountId`

Reply:
I've updated the keychain file to export functions both for generic password & password by service id. please check it again.